Rules.md

```diff
@@ -186,58 +182,51 @@ When a move proceeds into an enemy region, a battle is initiated with all moved
 A battle has one or more rounds. Every round has following parts:
 
 1. **Artillery strike:**
-    * Every healthy attacking artillery kills 1 unit (*)
+    * Every attacking artillery kills 1 unit (*)
 1. **Rolling dice:**
-    * Attacker gets a dice for every healthy attacking **infantry unit**
-    * Defender gets a dice for every healthy defending **unit** (infantry + artillery)
+    * Attacker gets a dice for every attacking **infantry unit**
+    * Defender gets a dice for every defending **unit** (infantry + artillery)
     * Every attacker roll of:
-        * **3-4:** Wounds an enemy unit (*)
-        * **5-6:** Kills an enemy unit (*)
+        * **3-4:** Wounds an enemy unit
+        * **5-6:** Kills an enemy unit
     * Every defender roll of:
-        * **1-4:** Wounds an enemy unit (*)
-        * **5-6:** Kills an enemy unit (*)
+        * **1-4:** Wounds an enemy unit
+        * **5-6:** Kills an enemy unit
 
-    Kills are resolved before wounds.
-1. **Attacker decides to retreat or has no healthy units left:** All units move back to the region
-   where to move started.
+   Wound triggers a retreat for the wounded unit(s).
+   Kills are resolved before wounds. Infantries are targeted before artillery.
+1. **Attacker retreat:** Wounded and other attacking units move back to the region where the move
```

Comment:
I purposely named it 'selected units' because I meant that each battle round you could choose to retreat with a subset of of your units on top of the wounded ones.
The way the rules are written now with you change to 'attacking' in stead of 'selected', they don't make sense, as the attacker would always need to retreat after the first round.